documentation/business-analyst-activities.md

```diff
@@ -1 +1,17 @@
 # Business Analyst Activities
+
+**How much detail should we go into with regards to setting up GITHUB account?
```

Comment:
I don't think we need to detail the process, but could say something like:
You must have a Github account and access to the Teaching Vacancies repository to be able to make any changes.

Also, thinking perhaps we should link to this from the README. It could be useful to direct any non devs here.

Reply:
Agree, a separate "Here's how non-technical team members can get set up and contribute on GitHub" doc might be useful to have, including details for the different permissions we have set up now.

Also it's **GitHub** #pedant 😉